flang/test/Lower/intrinsic-procedures/ubound.f90

```diff
@@ -2,86 +2,79 @@
-! CHECK:         %[[VAL_13:.*]] = fir.convert %[[VAL_12]]#1 : (index) -> i64
-! CHECK:         %[[VAL_14:.*]] = arith.constant 0 : index
-! CHECK:         %[[VAL_15:.*]] = fir.coordinate_of %[[VAL_3]], %[[VAL_14]] : (!fir.ref<!fir.array<2xi64>>, index) -> !fir.ref<i64>
-! CHECK:         %[[VAL_16:.*]] = fir.convert %[[VAL_5]] : (index) -> i64
-! CHECK:         fir.store %[[VAL_16]] to %[[VAL_15]] : !fir.ref<i64>
-! CHECK:         %[[VAL_17:.*]] = arith.constant 1 : index
-! CHECK:         %[[VAL_18:.*]] = fir.coordinate_of %[[VAL_3]], %[[VAL_17]] : (!fir.ref<!fir.array<2xi64>>, index) -> !fir.ref<i64>
-! CHECK:         %[[VAL_19:.*]] = fir.convert %[[VAL_7]] : (index) -> i64
-! CHECK:         fir.store %[[VAL_19]] to %[[VAL_18]] : !fir.ref<i64>
-! CHECK:         %[[VAL_20:.*]] = arith.constant 1 : i64
-! CHECK:         %[[VAL_21:.*]] = arith.subi %[[VAL_8]], %[[VAL_20]] : i64
-! CHECK:         %[[VAL_22:.*]] = fir.coordinate_of %[[VAL_3]], %[[VAL_21]] : (!fir.ref<!fir.array<2xi64>>, i64) -> !fir.ref<i64>
-! CHECK:         %[[VAL_23:.*]] = fir.load %[[VAL_22]] : !fir.ref<i64>
-! CHECK:         %[[VAL_24:.*]] = arith.constant 1 : i64
-! CHECK:         %[[VAL_25:.*]] = arith.subi %[[VAL_23]], %[[VAL_24]] : i64
-! CHECK:         %[[VAL_26:.*]] = arith.addi %[[VAL_25]], %[[VAL_13]] : i64
-! CHECK:         fir.store %[[VAL_26]] to %[[VAL_2]] : !fir.ref<i64>
+! CHECK:         %[[VAL_0:.*]] = fir.load %{{.*}} : !fir.ref<i64>
+! CHECK:         %[[VAL_1:.*]] = fir.address_of(
+! CHECK:         %[[VAL_2:.*]] = fir.convert %{{.*}} : (!fir.box<!fir.array<?x?xf32>>) -> !fir.box<none>
+! CHECK:         %[[VAL_3:.*]] = fir.convert %[[VAL_0]] : (i64) -> i32
+! CHECK:         %[[VAL_4:.*]] = fir.convert %[[VAL_1]]
+! CHECK:         %[[VAL_5:.*]] = fir.call @_FortranASizeDim(%[[VAL_2]], %[[VAL_3]], %[[VAL_4]], %{{.*}}) : (!fir.box<none>, i32, !fir.ref<i8>, i32) -> i64
+! CHECK:         %[[VAL_6:.*]] = fir.address_of(
+! CHECK:         %[[VAL_7:.*]] = fir.convert %{{.*}} : (!fir.box<!fir.array<?x?xf32>>) -> !fir.box<none>
+! CHECK:         %[[VAL_8:.*]] = fir.convert %[[VAL_0]] : (i64) -> i32
+! CHECK:         %[[VAL_9:.*]] = fir.convert %[[VAL_6]]
+! CHECK:         %[[VAL_10:.*]] = fir.call @_FortranALboundDim(%[[VAL_7]], %[[VAL_8]], %[[VAL_9]], %{{.*}}) : (!fir.box<none>, i32, !fir.ref<i8>, i32) -> i64
+! CHECK:         %[[VAL_11:.*]] = arith.subi %[[VAL_10]], %{{.*}} : i64
+! CHECK:         %[[VAL_12:.*]] = arith.addi %[[VAL_11]], %[[VAL_5]] : i64
+! CHECK:         fir.store %[[VAL_12]] to %{{.*}} : !fir.ref<i64>
   res = ubound(a, dim, 8)
 end subroutine
 
 ! CHECK-LABEL: func @_QPubound_test_3(
-! CHECK-SAME:  %[[VAL_0:.*]]: !fir.ref<!fir.array<10x20x?xf32>>,
-! CHECK-SAME:  %[[VAL_1:.*]]: !fir.ref<i64>,
-! CHECK-SAME:  %[[VAL_2:.*]]: !fir.ref<i64>) {
+! CHECK-SAME:  %[[ARG_0:.*]]: !fir.ref<!fir.array<10x20x?xf32>>,
+! CHECK-SAME:  %[[ARG_1:.*]]: !fir.ref<i64>,
+! CHECK-SAME:  %[[ARG_2:.*]]: !fir.ref<i64>) {
```

Comment:
You might have git conflicts here anyway, be beware that #1414 is impacting pretty much all test files and that you will want to make check-flang your PR once rebased.

Reply:
Yep.  Will do.